src/components/term_row.less

```diff
@@ -37,38 +37,22 @@
   }
 }
 
-.ansi-black-fg { color: #151515; }
-.ansi-red-fg { color: #fb9fb1; }
-.ansi-green-fg { color: #acc267; }
-.ansi-yellow-fg { color: #ddb26f;}
-.ansi-blue-fg { color: #6fc2ef; }
-.ansi-magenta-fg { color: #e1a3ee;}
-.ansi-cyan-fg { color: #12cfc0; }
-.ansi-white-fg { color: #d0d0d0; }
-
-.ansi-bright-black-fg { color: #505050; }
-.ansi-bright-red-fg { color: #fb9fb1; }
-.ansi-bright-green-fg {color: #acc267;}
-.ansi-bright-yellow-fg {color: #ddb26f;}
-.ansi-bright-blue-fg {color: #6fc2ef;}
-.ansi-bright-magenta-fg {color: #e1a3ee;}
-.ansi-bright-cyan-fg {color: #12cfc0; }
-.ansi-bright-white-fg { color: #f5f5f5; }
-
-.ansi-black-bg { background-color: #151515; }
-.ansi-red-bg { background-color: #fb9fb1; }
-.ansi-green-bg { background-color: #acc267; }
-.ansi-yellow-bg { background-color: #ddb26f;}
-.ansi-blue-bg { background-color: #6fc2ef; }
-.ansi-magenta-bg { background-color: #e1a3ee;}
-.ansi-cyan-bg { background-color: #12cfc0; }
-.ansi-white-bg { background-color: #d0d0d0; }
-
-.ansi-bright-black-bg { background-color: #505050; }
-.ansi-bright-red-bg { background-color: #fb9fb1; }
-.ansi-bright-green-bg {background-color: #acc267;}
-.ansi-bright-yellow-bg {background-color: #ddb26f;}
-.ansi-bright-blue-bg {background-color: #6fc2ef;}
-.ansi-bright-magenta-bg {background-color: #e1a3ee;}
-.ansi-bright-cyan-bg {background-color: #12cfc0; }
-.ansi-bright-white-bg { background-color: #f5f5f5; }
+.term-row .black     { color: black; }
+.term-row .red       { color: red; }
+.term-row .green     { color: green; }
+.term-row .yellow    { color: olive; }
+.term-row .blue      { color: blue; }
+.term-row .purple    { color: purple; }
+.term-row .cyan      { color: teal; }
+.term-row .white     { color: gray; }
+.term-row .bg-black  { background-color: black; }
+.term-row .bg-red    { background-color: red; }
+.term-row .bg-green  { background-color: green; }
+.term-row .bg-yellow { background-color: olive; }
+.term-row .bg-blue   { background-color: blue; }
+.term-row .bg-purple { background-color: purple; }
+.term-row .bg-cyan   { background-color: teal; }
+.term-row .bg-white  { background-color: gray; }
+.term-row .underline { text-decoration: underline; }
+.term-row .bold      { font-weight: bold; }
+.term-row .blink     { text-decoration: blink; }
```

Comment:
does `term_row.js` support bright colors? not sure it is a huge problem if they do not, but if so, would need the styles added